bokehjs/src/coffee/models/tools/gestures/pan_tool.coffee

```diff
@@ -39,44 +39,44 @@ class PanToolView extends GestureTool.View
     new_dx = dx - @last_dx
     new_dy = dy - @last_dy
 
-    hr = _.clone(frame.get('h_range'))
```

Comment:
It doesn't look like `hr` is mutated anywhere, so if it was ever needed in the first place, perhaps it is not needed now